src/index.js

```diff
@@ -81,6 +126,10 @@ export const render = async function render(path) {
   if (path.startsWith('/fragments')) {
     return renderFragment(path);
   }
+  // Handle AEM UE Pages
+  if (path.startsWith('/content')) {
```

Comment:
You probably don't want to include `/content/...` in the urls right?

I would suggest to request everything that is not /docs or /fragments from AEM.

Reply:
Addressed